src/components/Paginate.js

```diff
@@ -1,29 +1,112 @@
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
I was assuming that we'd either move all the files or turn this on repo-wide once we get there.

Reply:
@phulin yea, I think I just have a must more strict lint notifier... I can remove this if it's seen as noise.